nilearn/datasets/struct.py

```diff
@@ -1,6 +1,8 @@
 """
 Downloading NeuroImaging datasets: structural datasets
 """
+# Authors: Ana Luisa Pinho, Nicolas Gensollen, Jerome Dockes, Bertrand Thirion
```

Comment:
if you run git blame you will see many more authors of this file. these lines are always inaccurate / outdated anyway so I would suggest not adding it, the git history is a more reliable source